help/destinations/catalog/crm/microsoft-dynamics-365.md

```diff
@@ -0,0 +1,218 @@
+
+[!DNL Dynamics 365] uses OAuth 2 with Authorization Grant as the authentication mechanism to communicate with the [!DNL Contact Entity Reference API]. Instructions to authenticate to your [!DNL Dynamics 365] instance are further below, within [Authenticate to destination](#authenticate) section.
+
+## Use cases {#use-cases}
+
+As a marketer, you can deliver personalized experiences to your users, based on attributes from their Adobe Experience Platform profiles. You can build segments from your offline data and send these segments to [!DNL Dynamics 365], to display in the users’ feeds as soon as segments and profiles are updated in Adobe Experience Platform.
+
+## Prerequisites {#prerequisites}
+
+### Experience Platform prerequisites {#prerequisites-in-experience-platform}
+
+Before activating data to the [!DNL Dynamics 365] destination, you must have a [schema](/help/xdm/schema/composition.md), a [dataset](https://experienceleague.adobe.com/docs/platform-learn/tutorials/data-ingestion/create-datasets-and-ingest-data.html?lang=en), and [segments](https://experienceleague.adobe.com/docs/platform-learn/tutorials/segments/create-segments.html?lang=en) created in [!DNL Experience Platform].
+
+Refer to Adobe's documentation for [Segment Membership Details schema field group](/help/xdm/field-groups/profile/segmentation.md) if you need guidance on segment statuses.
+
+### [!DNL Microsoft Dynamics 365] prerequisites {#prerequisites-destination}
+
+Note the following prerequisites in [!DNL Dynamics 365], in order to export data from Platform to your [!DNL Dynamics 365] account:
+
+#### You need to have a [!DNL Microsoft Dynamics 365] account {#prerequisites-account}
+
+Go to the [!DNL Dynamics 365] [trial](https://dynamics.microsoft.com/en-us/dynamics-365-free-trial/) page to register and create an account, if you do not have one already.
+
+#### Create field within [!DNL Dynamics 365] {#prerequisites-custom-field}
+
+Create the custom field of type `Simple` with field data type as `Single Line of Text` which Experience Platform will use to update the segment status within [!DNL Dynamics 365].
+Refer to the [!DNL Dynamics 365] documentation to [create a field (attribute)](https://docs.microsoft.com/en-us/dynamics365/customerengagement/on-premises/customize/create-edit-fields?view=op-9-1) if you need additional guidance.
+
+An example setup within [!DNL Dynamics 365] is shown below:
+![Dynamics 365 UI screenshot showing the custom fields.](../../assets/catalog/crm/microsoft-dynamics-365/dynamics-365-fields.png)
+
+#### Register an application and application user within Azure Active Directory {#prerequisites-app-user}
+
+To enable [!DNL Dynamics 365] to access resources you will need to log in with your [!DNL Azure Account] to [[!DNL Azure Active Directory]](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#register-an-application-with-azure-ad-and-create-a-service-principal) and create the below:
```

Comment:
[...] and create the following:

- An Azure Active Directory application
- A service principal
- An application secret

Reply:
Also updated all occurrences of Azure AD > Azure Active Directory